metagpt/provider/bedrock/amazon_bedrock_api.py

```diff
@@ -0,0 +1,136 @@
+from typing import Literal
+import json
+from metagpt.const import USE_CONFIG_TIMEOUT
+from metagpt.provider.llm_provider_registry import register_provider
+from metagpt.configs.llm_config import LLMConfig, LLMType
+from metagpt.provider.base_llm import BaseLLM
+from metagpt.logs import log_llm_stream, logger
+from metagpt.provider.bedrock.bedrock_provider import get_provider
+from metagpt.provider.bedrock.utils import NOT_SUUPORT_STREAM_MODELS, get_max_tokens
+try:
+    import boto3
+    from botocore.eventstream import EventStream
+except ImportError:
+    raise ImportError(
+        "boto3 not found! please install it by `pip install boto3` ")
```

Comment:
if the independences and itself of boto3 is not huge, you can add it into requirements.txt.